forem/forem
# Update left sidebar links with new Crayons components

## What type of PR is this? (check all applicable)


## Description

This PR introduces [recently shipped buttons](https://github.com/forem/forem/pull/15311) and links to the left sidebar, similarly to what [has been done in the header](https://github.com/forem/forem/pull/15653) while ago.

There's one "significant" change in functionality though: we will no longer show "Follow" button in "Popular tags" in the left sidebar. Reason for this: it's only visible to anonymous users so it actually only triggers signup modal. But we already have at least 3 other prompts on home page for users to sign up + all other CTAs like "Save" in every feed.

## QA Instructions, Screenshots, Recordings

Click around home page and see if links in left sidebar display correctly.

You shouldn't really be able to spot any differences at first glance...

![image](https://user-images.githubusercontent.com/108287/146336983-d8ec974a-731e-4c14-8a71-5a8f13e2e132.png)


### UI accessibility concerns?

Hopefully this will actually improve accessibility.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
